store-frontend-broken-instrumented/frontend/app/views/spree/layouts/spree_application.html.erb

```diff
@@ -10,10 +10,25 @@
   type="text/javascript">
 </script>
 <script>
-  window.DD_RUM && window.DD_RUM.init({
+if (window.DD_RUM) {
+  window.DD_RUM.init({
     clientToken: '<%= ENV['DD_CLIENT_TOKEN'] %>',
     applicationId: '<%= ENV['DD_APPLICATION_ID'] %>',
+    sampleRate: 100,
+    trackInteractions: true,
+    service: 'storedog-ui',
+    env: 'production',
+    version: 1
   });
+
+  if (!window._DATADOG_SYNTHETICS_BROWSER) {
+    window.DD_RUM.setRumGlobalContext({'usr.handle': 'john@storedog.com'});
+  };
+
+  if (window.locatoin.href.includes('utm')) {
+    window.DD_RUM.setRumGlobalContext({'fromUtm': true});
```

Comment:
FYI, `window.DD_RUM.addRumGlobalContext('fromUtm', true});` otherwise `setRumGlobalContext` will remove 'fromUtm'. But here this won't get executed so we don't care.

Reply:
Got it, let's change it just to be safe